scene/gui/tree.cpp

```diff
@@ -2966,6 +3009,7 @@ int Tree::propagate_mouse_event(const Point2i &p_pos, int x_ofs, int y_ofs, int
 			col_width = MAX(button_w, MIN(limit_w, col_width));
 		}
 
+		// Interesting cell button detection code
```

Comment:
```suggestion
		// Interesting cell button detection code.
```
Also interesting how? What is this comment saying?